.github/scripts/release-ecr-tags.js

```diff
@@ -0,0 +1,21 @@
+/**
+ * Release Tags
+ *
+ * Creating release tag based on each release version for AWS ECR Public
+ *
+ */
+
+module.exports = ({ context }) => {
+  if (context.eventName === 'release') {
+    return getReleaseTag(context)
+  }
+  throw new Error('Release Violation: Could not determine the required release tags.')
+}
+
+function getReleaseTag(domain, app, context) {
```

Comment:
Actually, your code won't work.

`return getReleaseTag(context)` at line 10

pass in the first parameter which is used for `domain` but `domain` is never used in this function.

Hence your context will always be undefined.

You should get this runtime error
=> `TypeError: Cannot read property 'payload' of undefined`

Reply:
Great catch @jingyi2811, thanks!